qualang_tools/results/README.md

```diff
@@ -156,3 +156,56 @@ for i in range(len(freqs_external)):  # Loop over the LO frequencies
     # Process and plot the results
     ...
 ```
+
+
+## Data handler
+The `DataHandler` is used to easily save data once a measurement has been performed.
+It saves data into an automatically generated folder with folder structure:
+`{root_data_folder}/%Y-%m-%d/#{idx}_{name}_%H%M%S`.  
+- `root_data_folder` is the root folder for all data, defined once at the start
+- `%Y-%m-%d`: All datasets are first ordered by date
+- `{idx}`: Datasets are identified by an incrementer (starting at `#1`).  

```

Comment:
[imo]

Maybe it more convenient for users to have '{idx:04d}` in place of `{idx}`. within the folder it'll easier to sort.

Reply:
I'm undecided about this. I understand that the reason is so that it's easier to sort. However, I've tested on both Mac and Windows, and in both cases the file explorer sorts correctly.

IMO there's two downsides to specifying the number of zero-padded digits:
- What happens when the number of digits is exceeded? The default behaviour is that the number of digits increases
- Seeing `#0015_T1_experiment` doesn't look as nice as `#15_T1_experiment`

I therefore have a slight preference for not zero-padding, but I can be convinced if others have a strong opinion. Thoughts? @yomach @TheoLaudatQM

Reply:
Yes, the intention was to sort the folders.

> I've tested on both Mac and Windows, and in both cases the file explorer sorts correctly.

That's great! If sorting in the file explorer wouldn't be an issue, I think there's no need to pad indexes with 0s.

Reply:
I agree, I'm against zero padding. Most modern OSes have [natural sort order](https://en.wikipedia.org/wiki/Natural_sort_order)